Makefile

```diff
@@ -63,3 +64,11 @@ coverage:
 	@echo "Generating coverage report..."
 	$(PYTHON) $(TESTRUNNER) $(COVERAGE)
 	$(PYTHON) $(PLUGIN_TESTRUNNER) --device=default.qubit.autograd $(COVERAGE) --cov-append
+
+.PHONY:format
+format:
+ifdef check
+	black -l 100 ./pennylane ./tests --check
+else
+	black -l 100 ./pennylane ./tests
+endif
```

Comment:
Nice! Another thing can also be configuring a pre-commit hook to run `black` by default on the changed files. Let me know if this is of interest and can help with the configuration.

Reply:
I think this would be a very good idea to run `black` and even run tests associated to the changed files at the `pre-commit` hook. I can create a PR for this if you also think it's worth the try.